R/learner_survival_surv_parametric.R

```diff
@@ -137,14 +146,20 @@ LearnerSurvParametric = R6Class("LearnerSurvParametric",
 
     .predict = function(task) {
       pv = self$param_set$get_values(tags = "predict")
-      pred = invoke(.predict_survreg, object = self$model, task = task, learner = self, .args = pv)
+      if (pv$discrete) {
```

Comment:
this means the discrete parameter should have the required tag

Reply:
Yes, and it does right?
```
discrete = p_lgl(default = FALSE, tags = "predict")
```

Reply:
the "required" tag should be added

Reply:
ah, now added